core/runner.py

```diff
@@ -3,18 +3,18 @@
 import uuid
 import time
 import os
+import numpy as np
 
 from core.configuration import ConfigurationManager
 from custom.keras import model_from_serialized, get_optimizer
 from data.iterators import count_datapoints
 from data.iterators import create_random_train_dataset_iterator
 from data.iterators import create_random_test_dataset_iterator
 from core.utils.keras import train_keras_model, validate_keras_model
-from core.utils.keras import serialize_weights
+from core.utils.keras import serialize_weights, deserialize_weights
 from core.utils.dmlresult import DMLResult
 from core.utils.enums import JobTypes, callback_handler_no_default
-# from core.utils.federated_learning_utils import federated_averaging
-#       NOTE: Commented until the next PR is done.
+
```

Comment:
Nitpick: delete this extra line.

Reply:
Done

Reply:
If this is done, how come I still see the extra line -_- (not trying to be sassy)